### PR TITLE
docs: fix wrong description

### PR DESCRIPTION
**Pull request by ping40**
The original document doesn't match the code anymore.

### CODE REVIEW

**Comment by ping40** [comment]
@gyuho  retest didn't trigger build again  2 days ago, is there something wrong ? thanks 


the following error showed that the building progress was unstable.
    v3_grpc_test.go:1754: tls: failed to find any PEM data in certificate input
    v3_grpc_test.go:1754: tls: failed to find any PEM data in certificate input

**Comment by spzala** [comment]
@ping40 I have just tried starting the build process. Let's see how it goes :)

**Comment by spzala** [comment]
Yay build passed :) looks good but need someone else to review @bdarnell @xiang90

File: raft/raft_test.go [hunk 1]
```diff
@@ -616,22 +616,20 @@ func testLeaderElectionOverwriteNewerLogs(t *testing.T, preVote bool) {
 	}
 	// This network represents the results of the following sequence of
 	// events:
-	// - Node 1 won the election in term 1.
-	// - Node 1 replicated a log entry to node 2 but died before sending
-	//   it to other nodes.
-	// - Node 3 won the second election in term 2.
-	// - Node 3 wrote an entry to its logs but died without sending it
-	//   to any other nodes.
+	// - Node 1 starts the first election at term 1 and fails.
```
Comment (bdarnell) [inline raft/raft_test.go]: The old description was correct: it's telling how we could get to the configuration created by newNetworkWithConfig, not what is about to happen in this test.

Comment (spzala) [reply raft/raft_test.go]: @ping40 with comment from @bdarnell above I guess we should close this PR? Thanks!

**Pull request closed without merging**